api/internal/route.go

```diff
@@ -54,10 +55,13 @@ func SetUpRouter() *gin.Engine {
 	r := gin.New()
 	logger := log.GetLogger(log.AccessLog)
 	r.Use(filter.CORS(), filter.RequestId(), filter.IPFilter(), filter.RequestLogHandler(logger), filter.SchemaCheck(), filter.RecoverHandler())
-	r.Use(static.Serve("/", static.LocalFile(filepath.Join(conf.WorkDir, conf.WebDir), false)))
-	r.NoRoute(func(c *gin.Context) {
-		c.File(fmt.Sprintf("%s/index.html", filepath.Join(conf.WorkDir, conf.WebDir)))
-	})
+	filesystem := fs.FS(StaticFiles)
+	subtree, err := fs.Sub(filesystem, "html")
+
+	if err != nil {
+		log.Errorf("%s\n", err)
```

Comment:
I think we should panic here if we can't embed assets in API binary.

Reply:
Okay!! I thought users could opt to operate through CLI even without the UI, so skipped intentionally.
changing..